Tupfile

```diff
@@ -0,0 +1,3 @@
+include_rules
+
+: cv.tex	|> !xelatex	|> %B.pdf %B.out %B.aux %B.log
```

Comment:
how do you run the build with tup ?
can you add a line in the README for that ? 

thanks :dancer: